en/telemetry/microhard_serial.md

```diff
@@ -0,0 +1,35 @@
+# Microhard Serial Radio
+
+[Microhard Pico Serial](http://microhardcorp.com/P900.php) radios enable MAVLink communication between a radio on a vehicle and a GCS.  
+Microhard Pico Serial radios are 1 watt output radios that support point to point, point to multi-point, and mesh modes.
+A single ground station radio can be used to communicate with multiple vehicles using point to multi-point or mesh. Vehicles must have different IDs.
+
+![Microhard Radio](../../assets/hardware/telemetry/ark_microhard_serial.jpg)
+
+## Purchase:
+
+* [1W 900MHz Serial Telemetry Radio](https://arkelectron.com/product/1w-900mhz-serial-telemetry-air-radio/)
+* [1W 900MHz USB Serial Telemetry Radio](https://arkelectron.com/product/1w-900mhz-serial-telemetry-ground-radio/)
+* [1W 2.4GHz Serial Telemetry Radio](https://arkelectron.com/product/1w-2400mhz-serial-telemetry-radio/)
+* [1W 2.4GHz USB Serial Telemetry Radio](https://arkelectron.com/product/1w-2400mhz-usb-serial-telemetry-radio/)
+
+### Connecting
+
+Connection on the autopilot is done using a Pixhawk standard 6 pin JST GH Telemetry cable.
+When set to 100mW output power or less, power from the telemtry cable is sufficient.
+For higher radio output powers above 100mW, a battery connection is needed on the 2 Pin Molex Nano-Fit.
+
+Connection on the groundstation is done through USB C. 1W output powers are supported when using USB PD.
+
+## Setup/Configuration
+
+Using [Pico Config](https://arkelectron.com/wp-content/uploads/2020/09/PicoConfig-1.7.zip) on Windows, the radio settings can be adjusted.
+
+![Pico Config](../../assets/hardware/telemetry/pico_configurator.png)
```

Comment:
I think the steps are like this (please confirm):
1. Connect MH to a windows computer using USB-C. 
2. Run Pico Config. The two should connect and you will be able to edit the settings using PicoConfig
3. Do I set a baud rate at this point? If so, what value?
4. Anyway, I then need to create another connection, this time between QGC and the MH. So I add a new connection as per usual using the baud rate I set above.
5. Then I connect vehicle MH and flight controller via TELEM1. 
6. Then I can connect the vehicle unit to the windows PC using the FTDI adapter (ie to serial console). 
   - I presume I disconnect the unit already attached to the PC or QGC? Otherwise wont PicoConfig get confused about which unit to configure.
7. So then I match the baud rate on the vehicle to what I already set in QGC and for the ground unit.
8. Then I can disconnect PicoConfig. Both my ground and air units should have same baud rate and can talk. So does QGC. Everything good

If that is about right, then I might subedit this.

Reply:
3. You set the baud rate to the same baud rate used on the flight controller serial port.
6. Yes you need to disconnect the unit already attached to the PC.

Feel free to edit.

Reply:
Yes, but for 3 what "should" you use? What do YOU use? 

I ask because the default is pretty low https://docs.px4.io/master/en/peripherals/mavlink_peripherals.html#default-mavlink-ports and - 1200 B/s and the radios might be much more capable.

Reply:
Since the default baud rate on PX4 is 57600, I would go with that. 
No rush, I appreciate the work.

Reply:
I haven't done any tests personally, I always use 57600 baud for the serial link. There is also wireless link rate that defaults to 172800bps which corresponds to a 250kHz bandwidth. You need to have the wireless link rate set higher than the serial link rate. I think that might be too much information for this guide.

Reply:
@AlexKlimaj 

> I think that might be too much information for this guide.

The only requirement I have for the docs is that the product works and there is a decent turnkey setup provided for users. On top of that I hope that docs fairly position the product so that user can make an informed choice - ie as to the kinds of bandwidth and distance they can use the product, weight, power consumption, and tradeoffs for the above. Within that, I'm really happy for vendors to market the great things they are doing that differentiate their products. 

Upshot, if you think it will help differentiate the product and help people use it better, then providing additional configuration options and tradeoffs information at the end is completely OK :-). 

My impression here though would be that you would want to do some testing to kind of work out the tradeoffs.